packages/config/config/devices/0x0330/sr-zv9080a-a.json

```
@@ -0,0 +1,252 @@
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV9080A-A",
+	"description": "Curtain Motor Controller",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xD00D",
```

```suggestion
			"productId": "0xd00d",
```
Needs to be lowercase. I'll give you a more detailed review tomorrow.